aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -20,7 +20,393 @@ The model classes you create are known as POCO classes (from **P**lain **O**ld *
 
 In this tutorial, you write the model classes first, and EF Core creates the database.
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-5.0"
+
+## Add a data model class
+
+# [Visual Studio](#tab/visual-studio)
+
+Right-click the *Models* folder > **Add** > **Class**. Name the file *Movie.cs*.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+Add a file named *Movie.cs* to the *Models* folder.
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+Right-click the *Models* folder > **Add** > **New Class** > **Empty Class**. Name the file *Movie.cs*.
+
+---
+
+Update the *Movie.cs* file with the following code:
+
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Models/Movie.cs)]
+
+The `Movie` class contains an `Id` field, which is required by the database for the primary key.
+
+The <xref:System.ComponentModel.DataAnnotations.DataType> attribute on `ReleaseDate` specifies the type of the data (`Date`). With this attribute:
+
+* The user is not required to enter time information in the date field.
+* Only the date is displayed, not time information.
+
+[DataAnnotations](/dotnet/api/system.componentmodel.dataannotations) are covered in a later tutorial.
+
+## Add NuGet packages
+
+# [Visual Studio](#tab/visual-studio)
+
+From the **Tools** menu, select **NuGet Package Manager** > **Package Manager Console** (PMC).
+
+![PMC menu](~/tutorials/first-mvc-app/adding-model/_static/pmc.png)
+
+In the PMC, run the following command:
+
+```powershell
+Install-Package Microsoft.EntityFrameworkCore.SqlServer
+```
+
+The preceding command adds the EF Core SQL Server provider. The provider package installs the EF Core package as a dependency. Additional packages are installed automatically in the scaffolding step later in the tutorial.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+[!INCLUDE[](~/includes/add-EF-NuGet-SQLite-CLI-5.md)]
```

Comment:
This calls out the new targetFramework "net5.0" as I would expect.   However, later when I run dotnet aspnet-codegenerator I get an error indicating "Ensure that restore has run and that you have included 'net50' in the TargetFrameworks for your project. "   If I change the TargetFramework to "net50", it no longer gives me that error, but then instead I get a scaffolding failed error: Scaffolding failed.
"Please add Microsoft.VisualStudio.Web.CodeGeneration.Design package to the project as a NuGet package reference."    However I do have the latest added as a reference.   I assume you tested and it worked for you.  If so then disregard the issue I am running into.

Reply:
Hmm yeah that was an issue with older preview versions of the 5.0 sdk. Definitely have not run into something like that in a while. Are you testing using the rtm sdk?

Reply:
Thanks, I'll go back and check.  I had switched machines, which I thought were both up to date.

Reply:
@deepchoudhery, my apologies for the mistake.  My 2nd machine I had switched to was on 5.0.0-rc.2.20475.5.  I know better, sorry about that.

Reply:
@deepchoudhery  where can we get the RTM 5.0 SDK?

Reply:
> @deepchoudhery where can we get the RTM 5.0 SDK?

Got it, never mind.

Reply:
@Rick-Anderson, when you tested for VS Code, were you able to get past scaffolding fine?  I have been unable to with this PR plus the very latest .NET.  I may just have something messed up here that I'm not spotting.  If your test passed great.